Content.Shared/Research/Prototypes/TechnologyPrototype.cs

```diff
@@ -20,7 +20,13 @@ public sealed class TechnologyPrototype : IPrototype
         /// </summary>
         [ViewVariables]
         [DataField("name")]
-        public string Name { get; } = string.Empty;
+        public string Name
+        {
+            get => (_name is not null) ? Loc.GetString(_name) : ID;
+            private set => _name = Loc.GetString(value);
```

Comment:
Same as the other PR, you save it localised but also get it localised. You should just get it localised.

Reply:
Fixed. There was also duplication with the description field.